diff --git a/lib/lib.dom.d.ts b/lib/lib.dom.d.ts
index 4a1be1170b5f39cc844bdf0a1c5b89872380c40c..1fd652f50c115b4d97f668ef75fe0d05c83a8eb9 100644
--- a/lib/lib.dom.d.ts
+++ b/lib/lib.dom.d.ts
@@ -5344,6 +5344,7 @@ interface EventTarget {
     dispatchEvent(event: Event): boolean;
     /** Removes the event listener in target's event listener list with the same type, callback, and options. */
     removeEventListener(type: string, callback: EventListenerOrEventListenerObject | null, options?: EventListenerOptions | boolean): void;
+    value?: string
 }
 
 declare var EventTarget: {